### PR TITLE
Backend for simple blast prototype

### PR DESCRIPTION
**Pull request by JStinenbosch**

### CODE REVIEW

File: PythonFiles/Blast.py [hunk 1]
```diff
@@ -16,14 +16,14 @@ def runBlast():
     @staticmethod
     def make_blast_db(wgseqs, arguments):
```
Comment (FrankTN) [inline PythonFiles/Blast.py:17]: We should also add a way to specify the target path here. This should be done in such a way that the blastquery function knows where the db's are.

File: PythonFiles/BlastBuilder.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        if seq_type is "N":
+            self.WGS_flags = "nucl"
+        else:
+            self.WGS_flags = "prot"
+
+    def set_query_paths(self, path_list: list):
+        self.query_paths = path_list
+
+    def set_query_flags(self, seq_type: str):
+        if seq_type is "N":
```
Comment (FrankTN) [inline PythonFiles/BlastBuilder.py:25]: We can extract a function here to combine the behavior here with the behavior of set_WGS_flags

**Review by FrankTN — COMMENTED** [review]
We should still add some documentation in the more complicated functions, other than that it looks good

File: PythonFiles/Blast.py [hunk 3]
```diff
@@ -16,14 +16,14 @@ def runBlast():
...
         arg_string = ""
-        arg_string = add_property_to_string("dbtype", arguments, arg_string, "nucl")
+        arg_string = add_property_to_string("seq_type", arguments, arg_string, "nucl")
```
Comment (jurrdb) [inline PythonFiles/Blast.py]: revert seq_type to dbtype, since this is a blast argument.

File: PythonFiles/BlastBuilder.py (hunk 2 above)
Comment (jurrdb) [reply PythonFiles/BlastBuilder.py:25]: As of now, query flags are unused

**Review by jurrdb — CHANGES_REQUESTED** [review]

**Review by jurrdb — APPROVED** [review]

**Pull request merged**